### ISSUE TITLE
[ProjectTracking]: testing foo

### ISSUE THREAD

**Issue by akhi3030** [near/near-one-project-tracking#4]
# Goals

What is the project trying to achieve?  Why should the Near One team work on this project?  How does it improve the NEAR protocol?  etc.  Please be as explicit as possible in describing what needs to be achieved / accomplished in order for the project to be done.

# Links to external documentations and discussions

Please provide links to external discussions and documents such as google design docs; zulip design discussions; NEP discussions; documentation from other projects; etc.  Please take care that all these materials are publicly viewable when appropriate.

# Estimated effort

Please provide an estimate of which teams in Near One will need to contribute to this project and the estimated time to finish this project.  The estimated time should differentiate between engineering time and release time as appropriate.

Based on which teams from Near One will be involved, add appropriate `T-*` labels for the teams for them.

# Assumptions

Are there any assumptions that you are making that the stakeholders evaluating the viability of this project should be aware of?  Some examples are:
- If the project involves the validators to operate additional services; then we are hoping that the incentives will align that they will do it.

# Pre-requisites

What are the required things that need to be in place before this project can start?

# Out of scope

What works are not part of this project and will not be tackled?
